Features/Content/place.feature

```diff
@@ -11,3 +18,11 @@ Feature: Place
     Scenario: As a visitor, I can see the place list sorted
         Given I am on "/placelist/110/45.6152665/5.2234365/10000"
         Then I see "Mount Ventoux" text
+
+    Scenario: As a visitor, I can see the Mount Fuji page
+        Given I am on "/eZ-Mountains/Mount-Fuji"
+        Then I see "Mount Fuji" text
+        And I see "Mount Fuji, located on Honshu Island, is the highest mountain in Japan at 3,776.24 m (12,389 ft)." text
+        And I see the thumbnail for image with alternative text "mount fuji"
+        And Path for thumbnail image with alternative text "mount fuji" should match "/var/ezdemo_site/storage/images/media/images/mount-fuji/552-1-eng-GB/mount-Fuji_imageplacesize.jpg"
+        And File "/var/ezdemo_site/storage/images/media/images/mount-fuji/552-1-eng-GB/mount-Fuji_imageplacesize.jpg" exists
```

Comment:
No problem, but to be sure: can I use hard coded paths in `Context.php` like it is done in `pageIdentifierMap` variable? or should I totally avoid it?